console/network/src/testnet_v0.rs

```diff
@@ -165,12 +162,25 @@ impl Network for TestnetV0 {
     /// The function name for the inclusion circuit.
     const INCLUSION_FUNCTION_NAME: &'static str = MainnetV0::INCLUSION_FUNCTION_NAME;
     /// The maximum number of certificates in a batch.
+    #[cfg(any(test, feature = "test"))]
+    const MAX_CERTIFICATES: u16 = 25;
+    /// The maximum number of certificates in a batch.
+    #[cfg(not(any(test, feature = "test")))]
     const MAX_CERTIFICATES: u16 = 100;
-    /// The maximum number of certificates in a batch before consensus V3 rules apply.
-    const MAX_CERTIFICATES_BEFORE_V3: u16 = 100;
+    /// A list of (consensus_version, size) pairs indicating the maximum number of validators in a committee.
+    #[cfg(any(test, feature = "test"))]
+    const MAX_COMMITTEE_SIZE: [(ConsensusVersion, u16); 2] = [(ConsensusVersion::V1, 16), (ConsensusVersion::V3, 25)];
+    /// A list of (consensus_version, size) pairs indicating the maximum number of validators in a committee.
+    #[cfg(not(any(test, feature = "test")))]
+    const MAX_COMMITTEE_SIZE: [(ConsensusVersion, u16); 2] = [(ConsensusVersion::V1, 100), (ConsensusVersion::V3, 100)];
```

Comment:
What is the intention for adding this to the network trait?

It should be coupled with `MAX_CERTIFICATES`. And then `MAX_COMMITTEE_SIZE` is set using `N::MAX_CERTIFICATES`